swap_meet/clothing.py

```diff
@@ -1,2 +1,14 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+
+    def __init__(self, condition = 0):
+        super().__init__(condition = condition, category = "Clothing")
+        self.condition = condition
```

Comment:
Nice job  removing the category from the constructor, since we don't want to encourage the caller to try to change it. Instead, we pass the literal value that we want to set up to the parent's constructor

Since we're calling the parent's `__init__`, which already uses the `condition` value we pass in to set `self.condition`, we don't need to set it again ourself.

Also following the PEP8 guidelines around keyword and default values, we can rewrite this as

```python
    def __init__(self, condition=0):
        super().__init__(condition=condition, category="Clothing")
```